lib/krane/restart_task.rb

```diff
@@ -213,7 +213,7 @@ def patch_daemonset_with_restart(record)
     def delete_statefulset_pods(record)
       pods = kubeclient.get_pods(namespace: record.metadata.namespace)
       pods.select! do |pod|
-        pod.metadata.ownerReferences.find { |ref| ref.uid == record.metadata.uid }
+        pod.metadata&.ownerReferences&.find { |ref| ref.uid == record.metadata.uid }
```

Comment:
Fixes the `nil` errors encountered if bare pods are found in the namespaces. At Shopify nearly all app namespaces will have such a pod (e.g. secret seeder).

Reply:
🤦 Thank you for fixing that